pennylane/templates/subroutines/approx_time_evolution.py

```diff
@@ -140,6 +140,31 @@ def compute_decomposition(
 
         Returns:
             list[.Operator]: decomposition of the operator
+
+
+        .. code-block:: python
+
+            import pennylane as qml
+            from pennylane.templates import ApproxTimeEvolution
```

Comment:
```suggestion
            from pennylane import ApproxTimeEvolution
```

Reply:
Was also in the docstring for the op itself. Removed there as well.